src/glum/_glm.py

```diff
@@ -3127,10 +3127,12 @@ class GeneralizedLinearRegressor(GeneralizedLinearRegressorBase):
         set ``verbose`` to any positive number for verbosity.
 
     scale_predictors: bool, optional (default=False)
-        If ``True``, estimate a scaled model where all predictors have a
-        standard deviation of 1. This can result in better estimates if
-        predictors are on very different scales (for example, centimeters and
-        kilometers).
+        If ``True``, scale all predictors to have standard standard deviation one.
+        This can result in better estimates if predictors are on different scales
+        (for example, centimeters and kilometers). Should be set to ``True`` if
```

Comment:
Can we even remove this? "This can result in better estimates if predictors are on different scales (for example, centimeters and kilometers)." (Since, internally, the design matrix is scaled anyway?)